<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlimDisplay</class>
 <widget class="QMainWindow" name="FlimDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1397</width>
    <height>832</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FLIM Acquisition</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListView" name="file_list_view">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="spacing">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QMdiArea" name="mdi_area"/>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="options_tab">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Acquistion</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>9</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QGroupBox" name="groupBox_6">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="title">
              <string>Sequence</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label_28">
                <property name="text">
                 <string>Frames per Image</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="frame_accumulation_spin">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>1000</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Images</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="n_images_spin">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>1000000</number>
                </property>
                <property name="value">
                 <number>50</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Experiment Name</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Prefix</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="prefix_edit">
                <property name="text">
                 <string>Flim_Name</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Seq #</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="seq_number_spin">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>1000000</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Motion Correction</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="RealignmentParametersWidget" name="widget" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="live_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Live</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="acquire_sequence_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Acquire</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progress_bar">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stop_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="FlimStatusWidget" name="bh_rates_widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1397</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="open_workspace_action"/>
    <addaction name="separator"/>
    <addaction name="set_output_folder_action"/>
    <addaction name="separator"/>
    <addaction name="quit_action"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="tcspc_settings_action"/>
   </widget>
   <widget class="QMenu" name="menuSimulations">
    <property name="title">
     <string>Simulations</string>
    </property>
    <addaction name="generate_simulated_dataset_action"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuSettings"/>
   <addaction name="menuSimulations"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="quit_action">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="set_output_folder_action">
   <property name="text">
    <string>Set Auto Sequence Output Folder...</string>
   </property>
  </action>
  <action name="new_workspace_action">
   <property name="text">
    <string>New Workspace...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="open_workspace_action">
   <property name="text">
    <string>Open Workspace...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="tcspc_settings_action">
   <property name="text">
    <string>TCSPC Settings...</string>
   </property>
  </action>
  <action name="realignment_action">
   <property name="text">
    <string>Open Realignment Manager...</string>
   </property>
  </action>
  <action name="generate_simulated_dataset_action">
   <property name="text">
    <string>Generate Dataset...</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="actionShow_live_realignment_options">
   <property name="text">
    <string>Show live realignment options</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FlimStatusWidget</class>
   <extends>QWidget</extends>
   <header>FlimStatusWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RealignmentParametersWidget</class>
   <extends>QWidget</extends>
   <header>RealignmentParametersWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>file_list_view</tabstop>
  <tabstop>frame_accumulation_spin</tabstop>
  <tabstop>n_images_spin</tabstop>
  <tabstop>live_button</tabstop>
  <tabstop>acquire_sequence_button</tabstop>
  <tabstop>stop_button</tabstop>
 </tabstops>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
